apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -759,7 +759,7 @@ def append_obs(
     Writes new rows to an existing ``obs`` dataframe. (This is distinct from ``update_obs``
     which mutates the entirety of the ``obs`` dataframe, e.g. to add/remove columns.)
 
-    Example:
+    Example::
```

Comment:
Sphinx grossly misrenders this unless we have the double colon. Slightly uglier to have a double colon for `help(foo)` in Python; _much_ uglier to _not_ have the double colon for Sphix.